.github/workflows/push-changes.yaml

```diff
@@ -0,0 +1,25 @@
+name: Push Changes
+
+on:
+  push:
+    branches: [main]
+  workflow_run:
+    workflows:
+      - Cypress
+      - Docker
+    types:
+      - completed
+
+jobs:
+  dispatch:
+    strategy:
+      matrix:
+        repo: ['devfile/registry-support']
```

Comment:
you'd still need to update the registry-support workflow to receive it https://github.com/devfile/registry-support/blob/main/.github/workflows/pushimage-next.yaml